### PR TITLE
doc: add tips for build OS list

### PR DESCRIPTION
**Pull request by kindy**
##### Checklist

- [x] documentation is changed or added
- [x] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#commit-message-guidelines)

##### Affected core subsystem(s)

doc

### CODE REVIEW

**Review by gireeshpunathil — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ $ nvs use chakracore-nightly
 
 ### Building Node-ChakraCore
 
+Node-ChakraCore can be build on Wndows / Linux / Mac OS.
```
Comment (jackhorton) [inline README.md]: Windows needs to be spelled correctly. Also, this is a nitpick, but I feel like if we are going to say this, we should say exactly what kinds of linux and possibly minimum versions numbers for all three OSes.

**Review by jackhorton — CHANGES_REQUESTED** [review]
I don't think its a bad idea to state up front what OSes we support, but what motivated this change?

File: README.md (hunk 1 above)
Comment (kfarnung) [reply README.md]: I don't feel like this is necessary.  The sections below fulfill the same purpose.

**Review by kfarnung — CHANGES_REQUESTED** [review]

**Comment by kindy** [comment]
@jackhorton @kfarnung 

This is the reason that I add this line:
when I first time read this building section, I only see "Windows" on the screen.
then I think, maybe this is a windows-only project.

image

File: README.md [hunk 2]
```diff
@@ -63,6 +63,8 @@ $ nvs use chakracore-nightly
 
 ### Building Node-ChakraCore
 
+Node-ChakraCore can be build on Windows, Linux and Mac OS.
```
Comment (jackhorton) [inline README.md]: This should still be "Node-ChakraCore can be ***built*** on Windows, Linux, and ***macOS***"

**Review by jackhorton — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kindy) [reply README.md]: fixed.

**Review by kfarnung — APPROVED** [review]

**Comment by kfarnung** [comment]
@jackhorton Any issues with the current state of the PR?

**Review by jackhorton — APPROVED** [review]

**Comment by kfarnung** [comment]
Landed in bfc9ea165653f80bc80dcb6fd01404e9e8a12be6

**Pull request closed without merging**